addons/binding/org.openhab.binding.loxone/ESH-INF/thing/thing-types.xml

```diff
@@ -186,6 +186,12 @@
 		<label>Loxone Dimmer</label>
 		<description>Loxone's dimmer control</description>
 	</channel-type>
+	
```

Comment:
This needs to be indented with tabs, not spaces (requirement for xmls). You may use ctrl-shift-F under Eclipse to auto indent.